rpc/client/mock/client.go

```diff
@@ -76,61 +77,61 @@ func (c Call) GetResponse(args interface{}) (interface{}, error) {
 }
 
 func (c Client) Status() (*ctypes.ResultStatus, error) {
-	return core.Status()
+	return core.Status(&rpctypes.Context{})
```

Comment:
Maybe it would be better if we just initialized an empty context once in `c.ctx` and pass that in everywhere? Should be easy find/replace but might be cleaner?